apiExamples/outline.html

```diff
@@ -0,0 +1,4 @@
+<auro-tail tail="FI" outline></auro-tail>
```

Comment:
Might want to use a ligher tail color for demonstration, I could not even tell what `outline` did at first